src/BackupGenerator.php

```diff
@@ -0,0 +1,355 @@
+                        </a>
+                    </header>
+
+                    <section class="intro">
+                        <h1>Wallet Recovery Data Sheet</h1>
+                        <p>
+                            This document holds the information and instructions required for you to recover your Blocktrail wallet should anything happen. <br>
+                            Print it out and keep it in a safe location; if you lose these details you will never be able to recover your wallet.
+                        </p>
+                    </section>
+
+                    <section class="backup-info">
+                        <h2>Backup Info</h2>
+                        <div class="primary-mnemonic">
+                            <h3>Primary Mnemonic</h3>
+                            <code>$this->primaryMnemonic</code>
```

Comment:
use `"{$this->primaryMnemonic}"` with the curly braces, prevents excidental errors and for consistency (because you need if if it's anything other than a simple var eg `"{$this->method()}"`